source/include/dqm4hep/StatusCodes.h

```diff
@@ -40,7 +40,7 @@
   {                                                                                                                    \
     const dqm4hep::core::StatusCode statusCode(Command);                                                               \
     if (statusCode Operator StatusCode1) {                                                                             \
-      dqm_error("{0} return {1}, ", #Command, dqm4hep::core::statusCodeToString(statusCode));                          \
+      DQM4HEP_NO_EXCEPTION( dqm_error("{0} return {1}, ", #Command, dqm4hep::core::statusCodeToString(statusCode)); ); \
```

Comment:
Not sure what this means: Are you afraid that `dqm4hep::core::statusCodeToString` might throw?
And if it's the case now, what happens? Nothing is printed out?